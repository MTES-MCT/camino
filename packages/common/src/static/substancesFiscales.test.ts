import { isSubstanceFiscale, substancesFiscalesBySubstanceLegale } from './substancesFiscales'

test('isSubstanceFiscale', () => {
  expect(isSubstanceFiscale('wolf')).toBe(true)
  expect(isSubstanceFiscale('plop')).toBe(false)
})

test('substancesFiscalesBySubstanceLegale', () => {
  expect(substancesFiscalesBySubstanceLegale('auru')).toMatchSnapshot()
  expect(substancesFiscalesBySubstanceLegale('meba')).toMatchSnapshot()
})
