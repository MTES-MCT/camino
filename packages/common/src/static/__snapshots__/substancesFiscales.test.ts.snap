// Vitest Snapshot v1

exports[`substancesFiscalesBySubstanceLegale 1`] = `
[
  {
    "description": "contenu dans les minerais",
    "id": "auru",
    "nom": "or",
    "openFisca": {
      "nom": "aurifere",
      "unite": "mkg",
    },
    "substanceLegaleId": "auru",
    "uniteId": "mgr",
  },
]
`;

exports[`substancesFiscalesBySubstanceLegale 2`] = `
[
  {
    "description": "bauxite nettes livrées",
    "id": "aloh",
    "nom": "bauxite",
    "substanceLegaleId": "aloh",
    "uniteId": "mtk",
  },
  {
    "description": "contenu dans les minerais",
    "id": "cuiv",
    "nom": "cuivre",
    "substanceLegaleId": "cuiv",
    "uniteId": "mtt",
  },
  {
    "description": "contenu dans les minerais",
    "id": "etai",
    "nom": "étain",
    "openFisca": {
      "nom": "etain",
    },
    "substanceLegaleId": "etai",
    "uniteId": "mtt",
  },
  {
    "description": "net livré",
    "id": "fera",
    "nom": "pyrite de fer",
    "openFisca": {
      "nom": "fer_pyrite",
    },
    "substanceLegaleId": "ferx",
    "uniteId": "mtk",
  },
  {
    "description": "net livré",
    "id": "ferb",
    "nom": "minerais de fer",
    "openFisca": {
      "nom": "fer",
    },
    "substanceLegaleId": "ferx",
    "uniteId": "mtk",
  },
  {
    "description": "contenu dans les minerais",
    "id": "mang",
    "nom": "manganèse",
    "openFisca": {
      "nom": "manganese",
    },
    "substanceLegaleId": "mang",
    "uniteId": "mtc",
  },
  {
    "description": "contenu dans les minerais",
    "id": "plom",
    "nom": "plomb",
    "substanceLegaleId": "plom",
    "uniteId": "mtc",
  },
  {
    "description": "contenu dans les minerais",
    "id": "zinc",
    "nom": "zinc",
    "substanceLegaleId": "zinc",
    "uniteId": "mtc",
  },
]
`;
